lib/json_api_resource/resource.rb

```diff
@@ -72,6 +76,9 @@ def method_missing(method, *args, &block)
       else
         super
       end
+
+    rescue JsonApiClient::Errors::ServerError => e
+      pretty_error e
```

Comment:
json_api_client returns ServerError for every 500 series code from the server, so it should be OK.  I guess we really want to catch StandardError here and in querayble, but so far it has not been a problem.